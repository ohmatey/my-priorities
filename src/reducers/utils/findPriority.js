import { prioritiesList } from '../../data/'

const findPriority = id => prioritiesList.find(priority => priority.id === id)

export default findPriority
