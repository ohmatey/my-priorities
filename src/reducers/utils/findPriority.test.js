import findPriority from './findPriority'

describe('findPriority', () => {
  it('returns priority from id', () => {
    const foundPriority = findPriority(1)

    expect(foundPriority).toMatchSnapshot()
  })

  it('returns undefined with no id found', () => {
    const foundPriority = findPriority()

    expect(foundPriority).toMatchSnapshot()
  })
})
