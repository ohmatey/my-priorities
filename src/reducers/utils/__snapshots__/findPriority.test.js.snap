// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`findPriority returns priority from id 1`] = `
Object {
  "hours": 1,
  "id": 1,
  "name": "Yoga",
}
`;

exports[`findPriority returns undefined with no id found 1`] = `undefined`;
